prow/apis/prowjobs/v1/types.go

```diff
@@ -227,6 +227,7 @@ type ProwJobStatus struct {
 	State          ProwJobState `json:"state,omitempty"`
 	Description    string       `json:"description,omitempty"`
 	URL            string       `json:"url,omitempty"`
+	Path           string       `json:"path,omitempty"`
```

Comment:
Adding a top-level field to our API is a large change ... it at the very least requires some documentation